From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alistair Delva <adelva@google.com>
Date: Sun, 24 Jan 2021 17:22:51 -0800
Subject: ANDROID: x86: Add objtool to MAKE_GOALS

Prepare to enable LTO, which requires options that depends on objtool.

Bug: 175705438
Signed-off-by: Alistair Delva <adelva@google.com>
Change-Id: I830614180286b3e2222a5a445fb4db82658f6670
---
 build.config.x86_64 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build.config.x86_64 b/build.config.x86_64
--- a/build.config.x86_64
+++ b/build.config.x86_64
@@ -7,6 +7,7 @@ LINUX_GCC_CROSS_COMPILE_PREBUILTS_BIN=prebuilts/gas/linux-x86
 MAKE_GOALS="
 bzImage
 modules
+prepare-objtool
 "
 
 FILES="
